libs/ui/src/search_select.tsx

```diff
@@ -82,6 +82,7 @@ export interface SearchSelectMultiProps<T extends string = string>
   value: T[];
   onChange: (values: T[]) => void;
   disabled?: boolean;
+  menuPortalTarget?: HTMLElement;
```

Comment:
We might want to consider changing this everywhere, but I think this is the right decision for now. Maybe we track as a follow up

Reply:
yeah I didn't totally understand this change to be honest. It seemed to inexplictly make the font weight lighter on the selections in some vxsuite search select use cases

Reply:
https://github.com/votingworks/vxsuite/issues/6177 didn't assign anyone yet but feel free to assign yourself if you want to look into it